### PR TITLE
[migrations] added logs and air

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description
This change adds logs around the migration job as well as adding a bit of waiting between repeated db queries.

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] leeway-no-cache
      leeway-target=components:all
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-slow-database
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Comment by roboquat** [comment]
@svenefftinge: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: components/gitpod-db/src/long-running-migration/long-running-migration.ts [hunk 1]
```diff
@@ -42,39 +42,52 @@ export class LongRunningMigrationService {
...
+                        }
+                        if (migrationMetaData.completed) {
+                            continue;
+                        }
+                        log.info(`Running long running migration '${migration.getName()}' ...`);
+                        const now = new Date();
+                        try {
+                            const completed = await migration.runMigrationBatch();
+                            log.info(
+                                `Long running migration ${migration.getName()} took ${
```
Comment (easyCZ) [inline components/gitpod-db/src/long-running-migration/long-running-migration.ts:73]:
```suggestion
                                `Finished batch for migraiton ${migration.getName()} took ${
```

File: components/gitpod-db/src/long-running-migration/long-running-migration.ts [hunk 2]
```diff
@@ -42,39 +42,52 @@ export class LongRunningMigrationService {
...
+                            migrationMetaData = await repo.save({
+                                name: migration.getName(),
+                                firstRun: new Date(),
+                                lastRun: new Date(),
+                                completed: false,
+                            });
+                        }
+                        if (migrationMetaData.completed) {
+                            continue;
+                        }
```
Comment (easyCZ) [inline components/gitpod-db/src/long-running-migration/long-running-migration.ts:67]:
```suggestion
                        if (migrationMetaData.completed) {
                           log.info(`Migration ${migration.getName()} has completed`);
                            continue;
                        }
```

File: components/gitpod-db/src/long-running-migration/workspace-organizationid-migration.spec.db.ts [hunk 3]
```diff
@@ -52,7 +52,7 @@ describe("Workspace organizationid migration", () => {
             },
         });
 
-        const nowMinus30 = new Date(now.getTime() - 24 * 60 * 60 * 1000 * 30);
+        const nowMinus30 = new Date(now.getTime() - 24 * 60 * 60 * 1000);
```
Comment (easyCZ) [inline components/gitpod-db/src/long-running-migration/workspace-organizationid-migration.spec.db.ts:55]:
```suggestion
        const nowMinus1000 = new Date(now.getTime() - 24 * 60 * 60 * 1000);
```
? perhaps rename to "oldWorksapceTimestamp"

**Review by easyCZ — APPROVED** [review]

**Pull request merged**